### PR TITLE
Teig 4: Implemented dark mode, toggling of color mode and caching of color mode choice

### PR DESCRIPTION
**Pull request by AmirHassanConsulting**
Das [Pizza-Icon](https://github.com/SoulKa/teigrechner/compare/teig-4--toggle-color-mode?expand=1#diff-3c45b0358b8c6b8f4336762acae88c0a31d8a1bfd992c95a05b98603a09e4f74) ist mit [DALL-E](https://openai.com/index/dall-e-3/) generiert, daher liegt das Urheberrecht bei uns (siehe https://help.openai.com/en/articles/6425277-can-i-sell-images-i-create-with-dall-e).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -166,6 +169,33 @@ <h5>Zubereitung:</h5>
     }
   }
 
+  button {
```
Comment (SoulKa) [inline index.html]: this styles all buttons like this 🗡️

File: index.html [hunk 2]
```diff
@@ -166,6 +169,33 @@ <h5>Zubereitung:</h5>
...
+    background-color: transparent;
+    background-repeat: no-repeat;
+    border: none;
+    overflow: hidden;
+    outline: none;
+  }
+
+  .toggleColorScheme {
```
Comment (SoulKa) [inline index.html]: no camel case plz

File: main.js [hunk 3]
```diff
@@ -33,6 +38,10 @@ function init() {
 
   // Set random beer name
   document.getElementById('beer').textContent = getRandomBeer();
+
+  if(localStorage['colorMode'] == COLOR_MODES.DARK){
```
Comment (SoulKa) [inline main.js]: `===`

File: index.html [hunk 4]
```diff
@@ -166,6 +169,33 @@ <h5>Zubereitung:</h5>
...
+    height: 5%;
```
Comment (SoulKa) [inline index.html]: would rather use `em` instead of percent

File: index.html (hunk 1 above)
Comment (AmirHassanConsulting) [reply index.html]: Renamed to `.pizza-button`

File: index.html (hunk 2 above)
Comment (AmirHassanConsulting) [reply index.html]: Renamed to `.pizza-image`

**Review by SoulKa — APPROVED** [review]

**Pull request merged**